src/CodeStyle/Core/Analyzers/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,92 @@
+Microsoft.CodeAnalysis.Internal.Editing.DeclarationKind
```

Comment:
❗ This one should be reverted. We are using public API analyzer for a different purpose in this layer and these are not considered stable public APIs.